pandas/core/indexes/base.py

```diff
@@ -1552,12 +1552,19 @@ def droplevel(self, level=0):
 
         levnums = sorted(self._get_level_number(lev) for lev in level)[::-1]
 
-        if len(level) == 0:
+        return self._drop_level_nums(levnums)
+
+    def _drop_level_nums(self, levnums: List[int]):
```

Comment:
this returns an Index/MultiIndex right? (I know this code is all convoluted, but if can type pls do)

Reply:
tried that, mypy complained.  some Index methods need to be moved to MultiIndex to get tight annotations